### PR TITLE
Schedule for event, Issue #180

### PR DESCRIPTION
**Pull request by Mudit18**

### CODE REVIEW

**Comment by Mudit18** [comment]
@mananwason 
Please have a look.

**Comment by Mudit18** [comment]
Here are the screen shots for the schedule fragment.
device-2016-03-14-141503

If there are any issues please report.

File: app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.fossasia.openevent.data.Event;
+
+import java.util.ArrayList;
+
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleFragment extends Fragment {
+
+    private RecyclerView recyclerView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java]: Codacy Issue found: [Perhaps 'recyclerView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406672511/issues/source?bid=3233850&fileBranchId=3233902#l23)

File: app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package org.fossasia.openevent.adapters;
+
+import android.graphics.Color;
+import android.util.Log;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java]: Codacy Issue found: [Avoid unused imports such as 'android.util.Log'](https://www.codacy.com/app/triplez/open-event-android/file/2406672459/issues/source?bid=3233850&fileBranchId=3233902#l4)

File: app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleListAdapter extends BaseRVAdapter<Event,ViewHolder.Viewholder> {
+
+    public ScheduleListAdapter(List<Event> eventsList) {
+        super(eventsList);
+    }
+
+    private ViewHolder.SetOnClickListener listener;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/triplez/open-event-android/file/2406672459/issues/source?bid=3233850&fileBranchId=3233902#l26)

**Pull request closed without merging**